rasa/utils/endpoints.py

```diff
@@ -10,7 +10,7 @@
 from rasa.shared.exceptions import FileNotFoundException
 import rasa.shared.utils.io
 import rasa.utils.io
-from rasa.constants import DEFAULT_REQUEST_TIMEOUT
+from rasa.core.constants import DEFAULT_REQUEST_TIMEOUT
```

Comment:
This parameter won't be effected by the new CLI argument and will continue with the value initially assigned in constants file. Same with `rasa/core/actions/action.py` where requests to action server is made. Is this expected? @joejuzl @usc-m 

We may want to differentiate between these request timeouts? Another option is moving these variables from constants to a settings file and in there overriding the values from command lines? Semantically this would be better than changing inside "constants" file. Also they are not constants if they can be changed?

Reply:
The defaults are constants, it's just that we didn't have any way of making them non-default until recently it looks like?

Agree with the semantic confusion though - it is very confusing figuring out what should be affected by which setting especially as the names are very unclear

Reply:
It's not totally clear from the issue which timeouts are important to be able to change. Might be worth clarifying with the author.